config/prod.exs

```diff
@@ -0,0 +1,38 @@
+use Mix.Config
+
+config :maru, MongoosePush.Router,
+    versioning: [
+        using: :path
+    ],
+    https: [
+        ip: {127, 0, 0, 1},
```

Comment:
I don't think its a good idea. In many cases this project will be used on same host as `MongooseIM`. Also if somebody uses it on different host, he will notice that he has to change configuration because REST API won't work without it. But with `{0, 0, 0, 0}` user may not notice that his service is open for attacks, because it will always work this way, even if such configuration is not needed.